### ISSUE TITLE
SimpleIO will not convert path/uri parameters when no payload (body) is present

### ISSUE THREAD

**Issue by gaaf** [zatosource/zato#467]
A Service with Simple IO:
```
input_required = (Integer('number'),)
```

Will result in the following log entry when self.request/input is logged:
```
ServiceInput(number=u'12')
```

Which is a unicode wtring instead of an integer as specified. If in the same requets a payload is present, the input is correctly converted to integer.

**Comment by dsuch** [comment]
Looking into it now.

**Comment by dsuch** [comment]
Done in 1b07341678c97a9603545749b8761eeb59938096 - will be included in 2.0.6. Thank you @gaaf !
